i18n/en/docusaurus-plugin-content-docs/current/about/understanding/knowledge-types.md

```diff
@@ -3,28 +3,30 @@ sidebar_position: 3
 sidebar_label: Knowledge types
 ---
 
-# Types of knowledge in the project
+# Knowledge types in the project
 
-In any project, you can distinguish certain types of knowledge
+The following "types of knowledge" can be distinguished in any project:
 
-- **Fundamental knowledge**  
-  Knowledge that practically does not change over time.  
-  *Algorithms, computer science, the mechanisms of the YAP and its API, etc.*
+### Fundamental knowledge
```

Comment:
question: why did you decide to use headings instead of a list?

In my opinion, this page should not exist in the form that it currently exists in, and instead it should be a part of a larger page discussing the benefits of FSD. With that in mind, I'd actually prefer to shorten this page as much as possible so that it wouldn't occupy too much space when it's merged into another page.

Reply:
I think we can refactor itI think we can rewrite it when we paste it into another page. In general, lists with large items are harder to read.

Reply:
I agree about the large lists. However, I think we can approach this from another direction by reducing the text content (I'd argue, without losing too much informational value).

I took the liberty of experimenting and came up with this, what do you think about it?

<img width="803" alt="image" src="https://user-images.githubusercontent.com/15035286/215295960-90f38e1f-0e6c-4a97-b6d0-54de9b333705.png">

I changed it back into a list and reworded the description of the third point to be shorter

Reply:
Looks good

Reply:
Introduced the change, let me know if there's anything else you'd like to tweak, otherwise we can merge this